ow_sim_tests/test/sample_collection.py

```diff
@@ -17,6 +17,8 @@
 from geometry_msgs.msg import Point
 
 import owl_msgs.msg
+import ow_lander.constants
+import ow_lander.msg
 
 from action_testing import *
 
```

Comment:
Similar comment as the other test script. Lets remove `GROUND_POSITION` and use the ow_lander constant for `DISCARD_POSITION`. 

Lines 29-30 should become
```Python
DISCARD_POSITION = Point(1.5, 0.8, ow_lander.constants.DEFAULT_GROUND_HEIGHT) # default for discard action
```